var/spack/repos/builtin/packages/rccl/package.py

```diff
@@ -88,7 +90,7 @@ class Rccl(CMakePackage):
         "6.0.0",
         "6.0.2",
         "6.1.0",
-        "6.1.1",
+        "6.1.2",
     ]:
         depends_on(f"rocm-smi-lib@{ver}", when=f"@{ver}")
```

Comment:
Can you combine the 2 for loops into 1 inside the rccl package as both version list is same

Reply:
Removed redundant for loop